2996_reflection/reflection.md

```diff
@@ -1031,9 +1037,9 @@ Based on the [@N3980] API:
 ```cpp
 template <typename H, typename T> requires std::is_standard_layout_v<T>
 void hash_append(H& algo, T const& t) {
-    template for (constexpr auto mem : nonstatic_data_members_of(^T)) {
-        hash_append(algo, t.[:mem:]);
-    }
+  template for (constexpr auto mem : nonstatic_data_members_of(^T))
+    if (!(is_bitfield(mem) && !has_identifier(mem)))
+      hash_append(algo, t.[:mem:]);
```

Comment:
I'd rather keep the original instead of making the example more complicated.

Reply:
Reverted 👍